README.md

```diff
@@ -4,91 +4,56 @@
 
 Monika is a command line application for synthetic monitoring. The name Monika stands for "**Moni**toring Ber**ka**la", which means "periodic monitoring" in the Indonesian language.
 
-## Installation
+## Contributing
```

Comment:
Add How to use section, put link to documentation before this Contributing.

https://hyperjumptech.github.io/monika/overview